aws-lambdas/iac/terraform/main.tf

```diff
@@ -305,3 +330,70 @@ resource "aws_cloudwatch_event_target" "invoke_lambda" {
   rule = aws_cloudwatch_event_rule.lambda_cron.name
   arn  = module.cron.lambda_arn
 }
+
+################################################################################
+## elasticache-redis
+################################################################################
+
+module "vpc" {
```

Comment:
If you are creating new resources, you should be using the internal module for creating vpc and ec-subnets. 
* https://github.com/sourcefuse/terraform-aws-ref-arch-network

Reply:
not an issue anymore. not creating any vpc resources here.